src/index.ts

```diff
@@ -180,212 +155,208 @@ const getProfileFixture = (p?: Partial<Profile>): Profile =>
+        min: 3600,
+        max: 10000,
+      }) as NewMessageWithoutContent["timeToLiveSeconds"],
+      ...m,
+    }),
+    E.getOrElseW(err => {
+      throw new Error(`Cannot decode profile payload: ${PR.failure(err).join("\n")}`)
+    })
+  )
 
-const messageStatusCollectionUrl = documentDbUtils.getCollectionUri(
-  documentDbDatabaseUrl,
-  MESSAGE_STATUS_COLLECTION_NAME
-);
+const messageStatusCollection = dbInstance.container(MESSAGE_STATUS_COLLECTION_NAME)
 const messageStatusModel = new MessageStatusModel(
-  documentClient,
-  messageStatusCollectionUrl
+  messageStatusCollection
 );
 
-const getMessageStatusFixture = (ms?: Partial<MessageStatus>): MessageStatus =>
-  MessageStatus.decode({
-    messageId: ulid.ulid(),
-    status: faker.random.arrayElement([
-      MessageStatusValueEnum.ACCEPTED,
-      MessageStatusValueEnum.FAILED,
-      MessageStatusValueEnum.PROCESSED,
-      MessageStatusValueEnum.REJECTED,
-      MessageStatusValueEnum.THROTTLED,
-    ]),
-    updatedAt: faker.date.past(),
-    ...ms,
-  } as MessageStatus).getOrElseL((err) => {
-    throw new Error("Cannot decode message status payload:" + toString(err));
-  });
+const getMessageStatusFixture = (ms?: Partial<MessageStatus>): NewMessageStatus =>
+  pipe(
+    NewMessageStatus.decode({
+      messageId: ulid.ulid(),
+      status: faker.random.arrayElement([
+        MessageStatusValueEnum.ACCEPTED,
+        MessageStatusValueEnum.FAILED,
+        MessageStatusValueEnum.PROCESSED,
+        MessageStatusValueEnum.REJECTED,
+        MessageStatusValueEnum.THROTTLED,
+      ]),
+      updatedAt: faker.date.past(),
+      ...ms,
+    }),
+    E.getOrElseW(err => {
+      throw new Error(`Cannot decode message status payload: ${PR.failure(err).join("\n")}`)
+    })
+  )
 
-const notificationCollectionUrl = documentDbUtils.getCollectionUri(
-  documentDbDatabaseUrl,
-  NOTIFICATION_COLLECTION_NAME
-);
+const notificationCollection = dbInstance.container(NOTIFICATION_COLLECTION_NAME)
 const notificationModel = new NotificationModel(
-  documentClient,
-  notificationCollectionUrl
+  notificationCollection
 );
 
 const getNotificationFixture = (
   n?: Partial<NewNotification>
 ): NewNotification =>
-  NewNotification.decode({
-    id: ulid.ulid(),
-    fiscalCode: generateFakeFiscalCode(),
-    messageId: ulid.ulid(),
-    channels: {
-      EMAIL: {
-        addressSource: "PROFILE_ADDRESS",
-        toAddress: faker.internet.exampleEmail(),
-      },
-      WEBHOOK: {
-        url: "https://app-backend.io.italia.it/api/v1/notify?token=seceret",
+  pipe(
+    NewNotification.decode({
+      id: ulid.ulid(),
+      fiscalCode: generateFakeFiscalCode(),
+      messageId: ulid.ulid(),
+      channels: {
+        EMAIL: {
+          addressSource: "PROFILE_ADDRESS",
+          toAddress: faker.internet.exampleEmail(),
+        },
+        WEBHOOK: {
+          url: "https://app-backend.io.italia.it/api/v1/notify?token=seceret",
```

Comment:
small old typo: "secret".

Reply:
done in https://github.com/pagopa/io-fixtures/pull/12/commits/65b1baac5a94d63a31c81add5fc40b6d7134dec3